cmd/main.go

```diff
@@ -46,6 +46,8 @@ func main() {
 	cmd.Flags().StringVar(&diagParams.ECKVersion, "eck-version", "", "ECK version in use, will try to autodetect if not specified")
 	cmd.Flags().StringVar(&diagParams.OutputDir, "output-directory", "", "Path where to output diagnostic results")
 	cmd.Flags().StringVarP(&diagParams.OutputName, "output-name", "n", fmt.Sprintf("eck-diagnostics-%s.zip", time.Now().Format("2006-01-02T15-04-05")), "Name of the output diagnostics file")
+	cmd.Flags().StringVarP(&diagParams.ESUsername, "es-username", "u", "", "Optional Elasticsearch username for authentication (defaults to searching for credentials secrets in the resources namespace)")
+	cmd.Flags().StringVarP(&diagParams.ESPassword, "es-password", "p", "", "Optional Elasticsearch password for authentication (defaults to searching for credentials secrets in the resources namespace)")
```

Comment:
Why do we need this option? Will we not have always have a user with the necessary privileges available created by us?

Reply:
Yeah, that's why I was asking the question above. I added this option initially when we were just discussing being able to remove the elastic user, so this would require some file realm user for this to work. Since we're creating a dedicated diagnostics user now, I was questioning whether this was of any use. I'm guessing no. I'll get it removed.